.github/workflows/build_SRWA.yml

```diff
@@ -94,15 +94,32 @@ jobs:
     runs-on: self-hosted 
     if: ${{ github.event.label.name == 'ci-aws-intel-build' }}
     needs: [ Set_Repo_and_Branch, Clone_SRWA, Configure_SRWA, Build_SRWA ]
-    
+    outputs: 
+      build-success: ${{ steps.status.outputs.build-success }}
+
     steps:
       - name: Get build status
+        id: status
```

Comment:
Same comment for this file.  I don't understand why you didn't just use one job with multiple steps.  All we care about is whether or not the build was successful, we don't need/want status and granularity of each part of the build process.  What am I missing?